ui/src/store/resource.test.ts

```diff
@@ -573,6 +573,7 @@ describe('Store functions', () => {
               () => {
                 expect(resource.versions[1].minPipelinesVersion).toBe('0.12.1');
                 expect(resource.versions[1].version).toBe('0.1');
+                expect(resource.versions[1].versionPlatforms[1].name).toBe('linux/s390x');
```

Comment:
Can we have a test which fetches the platforms for latest versions and previous versions ??